test/test262-parser-tests/expectations.js

```diff
@@ -66,6 +55,14 @@ export default {
       // https://github.com/tc39/test262-parser-tests/issues/15
       '14eaa7e71c682461.js',
       'aca911e336954a5b.js',
+
+      // errors that michael disagrees with (function name `eval`/`arguments`/`package`/`static` invalid for strict mode function bodies)
```

Comment:
Is there an open issue for this? If not you should open one. This comment should refer to an issue, not a person.